src/api/comment/content-types/comment/lifecycles.js

```diff
@@ -1,5 +1,12 @@
 module.exports = {
 
+  beforeCreate(event) {
+    // Set author to the user sending the request
+    const ctx = strapi.requestContext.get();
+    event.params.data.author = ctx.state.user;
```

Comment:
what is the benefit of using this lifecycle, as opposed to setting the current user data on the front end which is the current implementation?

Reply:
It should be event.params.data.authorId = ctx.state.user. 
@dbradham the idea is to not allow someone pass arbitrary authorId from frontEnd, instead authorId is populated by backend using current user.

@oriyomibadmus Based on what you've discovered, if migrations are running before schema sync up, maybe there's no benefit on adding new column we can just migrate data into existing and change its type in schema sync. Could you please experiment with migrations locally?

Reply:
Thanks @becevka, but isn't this outside the scope of Remi's change? It looks more related to #158 and not his ticket. (https://app.zenhub.com/workspaces/-ideaspace-team-6350185d9843157427cc900b/issues/gh/dev-launchers/dev-launchers-platform/1684)

I'm confused on why this minor change is NOT approved, so Remi can continue to develop his other ticket? (https://app.zenhub.com/workspaces/-ideaspace-team-6350185d9843157427cc900b/issues/gh/dev-launchers/dev-launchers-platform/748)

Reply:
@becevka if you cannot approve this change, can you please schedule a call with me to discuss further? I am available any weeknight after 7pm Central US Time. Or on the weekends I can make myself available outside of 10-11am Central US Time.

Reply:
@dbradham Remi was interested in getting this code change through.
 The only note left is extra code which needs to be removed. 
 Also, this change kinda renders changes on Front End unnecessary as FrontEnd will not need to provide author.